web/src/components/Testimonial/Testimonial.tsx

```diff
@@ -43,7 +42,7 @@ export const Testimonial: React.FC<TestimonialProps> = (
 		{
 			[styles[`testimonial--${kebabCaseVariantClassName}`]]: variant,
```

Comment:
Turns out it was actually this line that is sometimes giving undefined, not className (I miscounted the props when I was looking 🤦‍♀️). Looks like this is undefined if we're on the default variant, can we update so its checking if variant is defined _and_ is not 'default'?